shared/domains/chelonia/internals.js

```diff
@@ -1143,21 +1143,23 @@ export default (sbp('sbp/selectors/register', {
       this.config.reactiveSet(state, contractID, Object.create(null))
       this.config.reactiveSet(state[contractID], '_volatile', currentVolatileState)
     }
-    const { HEAD: latestHEAD } = await sbp('chelonia/out/latestHEADInfo', contractID)
-    console.debug(`[chelonia] syncContract: ${contractID} latestHash is: ${latestHEAD}`)
-    // there is a chance two users are logged in to the same machine and must check their contracts before syncing
-    const { HEAD: recentHEAD, height: recentHeight } = state.contracts[contractID] || {}
-    const isSubcribed = this.subscriptionSet.has(contractID)
-    if (!isSubcribed) {
-      const entry = this.pending.find((entry) => entry?.contractID === contractID)
-      // we're syncing a contract for the first time, make sure to add to pending
-      // so that handleEvents knows to expect events from this contract
-      if (!entry) {
-        this.pending.push({ contractID })
-      }
-    }
-    this.postSyncOperations[contractID] = this.postSyncOperations[contractID] ?? Object.create(null)
+
```

Comment:
To be safe, could you move the `try` just above line 1132?

The reason I ask is because while we're making this change, it's a good idea to move it up because some of the lines above might throw. Specifically, `this.config.reactiveSet` could theoretically throw, and so could `sbp('okTurtles.events/emit', CONTRACT_IS_SYNCING, contractID, true)`